src/Components/SECard.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import '../Styles/SECard.css'
+
+function SECard({header1, image, alt,header2, para, header3, li1, li2, li3, li4}) {
+  return (
+    <div className='se-card-container'>
+      
+      <div className="se-card-content">
+
+        <div className="se-card-header1">
+          <p>{header1}</p>
+        </div>
+
+        <div className="se-card-icon">
+          <img src={image} alt={alt} />
+        </div>
+
+        <div className="se-card-header2">
+          <p>{header2}</p>
+        </div>
+
+        <div className="se-card-para">
+          <p>{para}</p>
+        </div>
+
+        <div className="se-card-header3">
+          <p>{header3}</p>
+        </div>
+
+        <div className="se-bullets">
+          <ul>
+            <li>{li1}</li>
+            <li>{li2}</li>
+            <li>{li3}</li>
+            <li>{li4}</li>
```

Comment:
Instead of passing these in individually, it could be passed in as an array that you map through. That would make this more generalized and able to accommodate any length of list items.

Reply:
Is there a way you can show us this ?

Reply:
@zcherries I will research it and see what I can figure out on this one. But I also agree with @Sinncity if you could so an example that would be amazing!